LEA_Footprint_Library.pretty/090329001.kicad_mod

```diff
@@ -0,0 +1,58 @@
+(footprint "090329001" (version 20221018) (generator pcbnew)
```

Comment:
it is not clear what the dot is showing
![image](https://github.com/upb-lea/LEA_KiCad_Library/assets/62549000/1ba04ec4-ab1d-411b-b96b-34e1ba7a16c4)